pkg/pillar/pubsub/pubsub.go

```diff
@@ -1159,6 +1162,7 @@ func handleModify(ctxArg interface{}, key string, item interface{}) {
 	// NOTE: without a deepCopy we would just save a pointer since
 	// item is a pointer. That would cause failures.
 	newItem := deepCopy(item)
+	create := false
```

Comment:
Nit, but "create" sounds like an action you would do, while "created" or "added" would be more accurate here.